src/pages/detail/tabs/members/GroupMembersTab.tsx

```
@@ -1,23 +1,35 @@
+  if (!members){
+    return <p>Loading members...</p>
+  }
+  
   return <div className="flex flex-col items-center">
-    <div className={"flex items-center justify-between my-6 w-2/3 rounded-2xl bg-greyLight px-5 py-[15px] text-lg text-greyDark"}>
+    {/* <UserCircle></UserCircle> */}
+    <div className={"text-[20px] text-greyDark"}>{group.memberCount}명</div>
+
+    {members.map(({name,email, role}:{name: string, email: string, role: string})=> 
+    <div className={"flex items-center justify-between my-6 w-1/2 rounded-2xl bg-greyLight px-5 py-[15px] text-lg "}>
       <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)